### DISCUSSION TITLE
Linking Scenes

### DISCUSSION THREAD

**Discussion by federicoacker** [MeusRex/pf2e-chex#10] [Ideas]
Might be cool to be able to link a scene to a hex, so when the players travel to the hex they would get a popup asking them if they wish to travel to the linked scene. It would be a nice added feature for tiles that have for examples dungeons or mines or fortresses

**Comment by MeusRex** [comment]
Hmm, might be interesting. I was thinking about adding a journal, but this might even be better. 
You could add a city map for towns, and cave maps for mines. Ya, I'll check how easy it is to move players to a different scene. I have like the echo of a memory that only the GM can move people around. If that's the case this would get a little more complicated since I will have to mess with event hooks to notify the GM's client that he should move a player to a different scene.
But it should be certainly doable, good idea.

**Reply by federicoacker** [reply]
You should check how modules like monk's tile triggers and stairways (teleporter) do it, in both cases players can just click on the tile/icon and get teleported to a new scene

**Reply by MeusRex** [reply]
Ya, multilevel tokens also support it. The issue there is that both of these coders love their 2400 lines, javascript monolith files with little organization as far as I can tell :D

**Reply by federicoacker** [reply]
Yikes, talk about readability hahahah
